{
 "errors": {},
 "symbols": [
  "pytest_catchlog.CallableList",
  "pytest_catchlog.CallablePropertyMixin",
  "pytest_catchlog.CallableStr",
  "pytest_catchlog.CatchLogPlugin",
  "pytest_catchlog.CompatLogCaptureFixture",
  "pytest_catchlog.LogCaptureFixture",
  "pytest_catchlog.LogCaptureHandler",
  "pytest_catchlog.__version__",
  "pytest_catchlog.add_option_ini",
  "pytest_catchlog.caplog",
  "pytest_catchlog.capturelog",
  "pytest_catchlog.catching_logs",
  "pytest_catchlog.get_logger_obj",
  "pytest_catchlog.get_option_ini",
  "pytest_catchlog.logging_at_level",
  "pytest_catchlog.logging_using_handler",
  "pytest_catchlog.pytest_addoption",
  "pytest_catchlog.pytest_configure"
 ]
}